Modules/_bz2module.c

```diff
@@ -277,7 +277,7 @@ BZ2_Malloc(void* ctx, int items, int size)
 {
     if (items < 0 || size < 0)
```

Comment:
Wouldn't a better change be to change this line to `size <= 0`? That way, you're only concerned out size's value in one place, instead of two.

Reply:
The behavior would be different. `_PyMem_RawMalloc` allocates memory even if `size == 0` (See [Objects/obmalloc.c](https://github.com/python/cpython/blob/6015cc50bc38b9e920ce4986ee10658eaa14f561/Objects/obmalloc.c#L79)).

Reply:
Ah, my mistake. I misread your test as using `||`, not `&&`. And I guess I misread the `!=`, too. I should have taken more time. My apologies.

So, this change looks good to me. You should add a news entry describing the change.